src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -344,7 +344,7 @@ def update_project_uptime_subscription(
                 case ObjectStatus.DISABLED:
                     disable_uptime_detector(detector)
                 case ObjectStatus.ACTIVE:
-                    enable_uptime_detector(detector)
+                    enable_uptime_detector(detector, ensure_assignment=True)
```

Comment:
Should we pass `ensure_assignment` as a parameter to `update_project_uptime_subscription`, so that we only set it when we're going from onboarding -> active?

Right now, we're going to be forcing assignment whenever we update an uptime monitor at all